File: 1.js
//If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.

//Find the sum of all the multiples of 3 or 5 below 1000.

function multi(num){
  var num_arr = []
  for (i = 1; i < num; i++)
    if (i % 3 === 0){
      num_arr.push(i);
    } else if (i % 5 === 0) {
      num_arr.push(i);
    }
  var sum = num_arr.reduce(function(x,y) {return x + y;}, 0);
  console.log(sum);
}

multi(10);

File: python_solutions/1.py




def multi(num):
    total = 0

    num_range = range(0,num)

    for x in num_range:
        if x % 3 == 0 or x % 5 == 0:
            total = total + x

    print total


    # while i < 0:
    #     if i % 3 === 0:
    #         num_arr.append(i)
    #     elif i % 5 === 0:
    #         num_arr.append(i)
    #     i++
    #
    # for multiple in num_arr:
    #     print multiples

multi(1000)
